src/sdk/resources/transferItems.ts

```diff
@@ -16,11 +16,23 @@ export class TransferItemsResource extends PagedResource<TransferItem> {
 		return this.modifyItem(itemid, {retry: true}, token);
 	}
 
+	ignoreMultiple(itemids?: any, token?: CancellationToken): Promise<void> {
```

Comment:
this is restricting us to providing a list of ids.  would it make more sense accept params here to allow for different search options?